src/ngx_http_lua_common.h

```diff
@@ -107,6 +107,9 @@ typedef struct ngx_http_lua_main_conf_s  ngx_http_lua_main_conf_t;
 typedef struct ngx_http_lua_srv_conf_s  ngx_http_lua_srv_conf_t;
 
 
+typedef struct ngx_http_lua_balancer_peer_data_s ngx_http_lua_balancer_peer_data_t;
```

Comment:
Style: two spaces needed before the type being defined (i.e., before `ngx_http_lua_balancer_peer_data_t`).